components/rmrk/Gallery/CuratedList.vue

```diff
@@ -122,6 +127,18 @@ export default class CuratedList extends mixins(AuthMixin) {
     }
   }
 
+  @media screen and (max-width: 368px) {
+    .title.is-5 {
+      font-size: medium;
+    }
+  }
+
+  @media screen and (max-width: 320px) {
+    .title.is-5 {
+      font-size: small;
+    }
+  }
+
```

Comment:
Not a fan of this solution with multiple breakpoints can you try to use `em` values instead of absolute size?

Reply:
added the 65% width at touch screens as you suggested, imo no need to make the font smaller anymore  
  
![image](https://user-images.githubusercontent.com/73316633/166226161-b4478db3-c47d-4ff1-b2c4-379cca9fcfe8.png)

Reply:
Noice!